Sources/AccessibilitySnapshot/Core/Swift/Classes/UIAccessibility+SnapshotAdditions.swift

```diff
@@ -519,6 +561,12 @@ extension UIAccessibilityTraits {
 
     static let switchButton = UIAccessibilityTraits(rawValue: 0x0020000000000000)
 
+    static let isEditing = UIAccessibilityTraits(rawValue: 0x0000000000200000)
+
+    static let textEntry = UIAccessibilityTraits(rawValue: 0x0000000000040000)
```

Comment:
I've seen references [here](http://www.openradar.me/35091280) and [here](https://gist.github.com/AndreasVerhoeven/65925ebb1a648c55e7a5dc4730c06be8) about these 2 private traits, I could find anything related to the `scrollable` one. Basically the traits are
`UITextField` = `textEntry` | `isEditing` (when editing)
`UITextView` = `textEntry` | `scrollable` | `isEditing` (when editing)